src/Handlers/MainContainer.cpp

```diff
@@ -19,7 +22,8 @@ int MainContainer::Init () {
 			c.append_attribute("cross").set_value("GPU");
 		#endif
 		solver->configfile.save_file(filename);
```

Comment:
BUT we don't want to process altered XML. 
I might add save of deep copy instead of solver->configfile directly